src/components/Home.vue

```diff
@@ -1,22 +1,24 @@
 <template>
   <div class="home">
-    <div class="head">
-      <h1>Your Courses </h1>
-    </div>
-    <div class="head">
-      <h1>{{semester}}</h1>
-    </div>
-    <el-card class="box-card" v-for="a in TD" :key="a">
-      <div slot="header" class="card-title" style="float: left">
-       <span>{{a.name}}</span>
-      </div>
-      <div class="text item">
-        <span>{{a.address}}</span>
-      </div>
-      <div class="bottom" style="float: left">
-        <span>状态栏</span>
-      </div>
-    </el-card>
+    <el-row style="float: left;margin: 4% 0 10px 0;width: 100%">
+        <span style="font-size: 30px">Your Courses</span>
+    </el-row>
+    <el-row style="float: left;margin: 4% 0 10px 0;width: 100%">
+       <span style="font-size: 20px">{{ semester }}</span>
+    </el-row>
+       <el-card class="box-card" v-for="a in TD" :key="a" style="height: 250px;width: 400px;margin: 3% 5% 5% 0">
```

Comment:
your can use :key="a.name"
and the variable name TD seems too difficult for others to understand what it means